release/src/main/scripts/build_release_candidate.sh

```diff
@@ -139,7 +139,7 @@ fi
 
 
 RC_TAG="v${RELEASE}-RC${RC_NUM}"
-USER_REMOTE_URL=git@github.com:${USER_GITHUB_ID}/beam-site
+USER_REMOTE_URL=https://${USER_GITHUB_ID}@github.com/${USER_GITHUB_ID}/beam-site.git
```

Comment:
The @ part doesn't look right, shouldn't it be something like https://github.com/ibzib/beam-site ?

Reply:
I think so but this worked for me so keeping it as it is.